source/docs/3/advanced_raster_analysis.rst

```diff
@@ -0,0 +1,176 @@
+Advanced Raster Analysis (QGIS3)
+=================================
+
+For city planning, the most useful raster data is Land-Cover. This data informs a wide variety of strategic planning activities. To do this, QGIS has advanced raster capabilities built-in via the ``Raster Calculator``. In this tutorial, we will explore the options available for styling categorical raster and the functionality provided by the raster calculator.
+
+Overview of the task
+--------------------
+
+We will identify the suitable areas for development by monitoring change/class transition in the city of Johannesburg, South Africa. 
+
+Other skills you will learn
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- How to load an excel file in QGIS.  
+- How to view a particular class in the raster layer.
+
+Get the data
+------------
+
+- We will use `The South African National Land Cover 2018 (SANLC) <https://www.environment.gov.za/projectsprogrammes/egis_landcover_datasets>`_ data, and the  *Land Cover Change 2014-18* data. 
```

Comment:
You need to add the link to the download site. Also add steps to accept terms, create a login, find the dataset link on the page, and show which datasets need to be downloaded. Then mention in 1 line what tool can be used to clip and reference the tutorial for it.